### PR TITLE
Implement contemporary dnn Buttons with color names

### LINKED ISSUE #288: Names and purpose of the colors
## Please summarize your question in one sentence
Is there an explanation which color variables are used fir what purpose?

## Give a more extended description
The color variables have names. Which is intended for what?

## Other comments or remarks

### PR DESCRIPTION
**Pull request by EPTamminga**
## Related to Issue
Fixes #288 #286 

## Description
Added a few color names, added explanation for other color names
Changed the _dnn button definitions to create flat contemporary-looking buttons

## How Has This Been Tested?
Used a HTML text module with all different kinds of HTML elements

## Screenshots (if appropriate):
image

## Types of changes
Colors, DNN button scss

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by EPTamminga** [comment]
It would be nice if the active colors could be calculated automatically from the non-active background colors. I do not know enough about the mixins functions in the nvQT to handle that.
Please advise.

File: src/scss/variables/_colors.scss [hunk 1]
```diff
@@ -1,17 +1,20 @@
 // Color Array
 $colors: (
-    "white":white,
-    "light-shade":#e6e6e6,
+    "white":white,              // Menu button font color
```
Comment (david-poindexter) [inline src/scss/variables/_colors.scss:3]:
```suggestion
    "white":white,
```
Removing comment, as this is not the only place this is, or can be, used.

File: src/scss/variables/_colors.scss [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
+    "light-shade":#e6e6e6,      // Menu row background color
```
Comment (david-poindexter) [inline src/scss/variables/_colors.scss:4]:
```suggestion
    "light-shade":#e6e6e6,
```
Removing comment, as this is not the only place this is, or can be, used.

File: src/scss/variables/_colors.scss [hunk 3]
```diff
@@ -1,17 +1,20 @@
...
     "light-accent":#e0e1eb,
-    "main-shade":#ec3d46,
+    "main-shade":#ec3d46,       // Menu button background color and page footer color
```
Comment (david-poindexter) [inline src/scss/variables/_colors.scss:6]:
```suggestion
    "main-shade":#ec3d46,
```
Removing comment, as this is not the only place this is, or can be, used.

**Review by david-poindexter — APPROVED** [review]
@EPTamminga thank you for the PR!  There are some great enhancements here that many will find useful.  🎉 

Also, it dawned on me during this review that we have some issues with the actual color palette here and it does not match the color palette currently used on https://nvquicktheme.com.  We will work to resolve this prior to the next release.

**Pull request merged**